meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
+# TODO fix this
```

Comment:
Hi, Derek.  Is this comment here because the version is hard coded to 6.8, or is there some other thing that needs fixing?

Thanks!